src/parquet/column/properties.h

```diff
@@ -162,14 +150,13 @@ class PARQUET_EXPORT WriterProperties {
     }
 
     std::shared_ptr<WriterProperties> build() {
-      return std::shared_ptr<WriterProperties>(new WriterProperties(allocator_,
-          dictionary_enabled_, dictionary_pagesize_, default_encoding_, encodings_,
-          pagesize_, version_, default_codec_, codecs_));
+      return std::shared_ptr<WriterProperties>(
+          new WriterProperties(allocator_, dictionary_pagesize_, default_encoding_,
+              encodings_, pagesize_, version_, default_codec_, codecs_));
     }
 
    private:
     MemoryAllocator* allocator_;
-    bool dictionary_enabled_;
```

Comment:
Based also on the discussions in PARQUET-573 this will need to be restored either now or later to a per-column on/off state (and if the the writer "exits" dictionary encoding due to the dictionary growing too large). I'm ok with punting on this here and tackling in a follow up PR or in #143)